docker/README.md

```diff
@@ -67,7 +67,7 @@ docker rm -f mms
 ```
 
 ## Available pre-built continers
-We have 5 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
+We have 8 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
```

Comment:
I see only 7 tags on Docker hub. 
Am I missing some tag ? 
https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/tags

Reply:
Good catch, @piyushghai ! @vdantu let's please make sure to match our document with what we currently have in dockerhub.

Reply:
I don't think we should touch this document for every release of MMS. I have a couple of lines there calling out that we have one released container for every release of MMS since 1.0.0